src/utils/formValidator.test.js

```diff
@@ -0,0 +1,7 @@
+import { checkAllEmptyInputs } from './formValidator';
+
```

Comment:
There should be tests for `validateInput` and `fieldChecker` as well.

Reply:
Done